### ISSUE TITLE
Missing Adium .SVG logo

### ISSUE THREAD

**Issue by Neustradamus** [adium/adium#3]
I do not found the Adium .SVG logo, where it is?

If it does not exist, can you create?

Thanks in advance.

**Comment by mathuaerknedam** [comment]
To the best of my knowledge, an SVG version of the Adium logo has never been created. I do have vector art of some sort (can't open right now to see) from the Vectory Ducks Xtras (https://www.adiumxtras.com/index.php?a=xtras&xtra_id=2222), but these are not the official Adium logo, and are not authorized for use as the official Adium logo. I'm not up on the current state of vectorization tech, but it might not be too hard to convert the old pixel art into vectors with decent results.

**Comment by wixardy** [comment]
[Here's a .psd](https://hg.adium.im/www/adium.im/raw-file/tip/images/adiumy/AdiumyGreen-512x512-200ppi.psd) which you can probably create an SVG from.

**Comment by donatj** [comment]
That PSD doesn't appear to have any layer information, it's just a flat bitmap.

**Comment by RobFog** [comment]
@Neustradamus What do you need it for?

**Comment by Neustradamus** [comment]
Thanks all for your comments!

I think it is time to create an official .SVG logo :)

In 2020, all good projects (and organizations/companies) have his logo in .SVG and it is really better to have an official file that unofficial ^^

**Comment by jas8522** [comment]
@RobFog who made the icon you're using here? That seems like it was created as a vector image in that it has more simplified gradients/shading and cleaner lines. Perhaps we could ask them if it could be the official SVG version of the logo.

**Comment by RobFog** [comment]
I don't know. I only recall the original one was by Adam Betts. Maybe @wixardy knows?

By the way, I'm not convinced "all good projects have it" is a reason that is worth the effort. If that is actually the case, @Neustradamus surely should be able to give us a proper reason?

**Comment by jas8522** [comment]
@RobFog agreed.

The image you're using is from the xtras set @mathuaerknedam [mentions above](https://www.adiumxtras.com/index.php?a=xtras&xtra_id=2222). Shall we reach out to Sam Gwilym about making it an official vector of the logo? I'm a fan. I think it does an excellent job of capturing the animation style while simplifying the overall appearance.

I should add that I've converted bitmaps to svg's before and tried to do this with the Adiumy logo, however it's got far too many shadowing and gradient layers to be done easily. Sam's version would be a much better approach, if they're willing to authorize its use.

**Comment by jas8522** [comment]
@sgwilym I think this is your Icon! Thoughts?

**Comment by sgwilym** [comment]
@jas8522 Yes, I made these! I was very surprised to get a message about these icons after all these years.

Whatever you decide upon, I'd be delighted to donate these icons to the Adium project.

One thing I'll say is I think my younger self wasn't able to appreciate the grizzled charm of the original icon. Seeing it now I find it rather nostalgic and charming. :)

**Comment by RobFog** [comment]
I've used them a lot. Thank you for making, them, @sgwilym. 🙂

I'm still unclear as to what the purpose of bundling an .svg version of the app logo is. If the purpose is for people to be able to manipulate the official logo then I'm skeptical whether bundling a modified version is a good idea. It's potentially misleading and waters down the branding.

**Comment by jas8522** [comment]
Personally, I'd consider this thread a reasonable resource if someone went looking for an SVG version of the logo and ended up using Sam's. Therefore IMO simply closing this and having it on record is sufficient. As in, I don't know if it explicitly needs to be bundled with the app. Might be nice to include it either on the website or GitHub page; but I'll leave that to consensus. Other opinions I'm sure are to come 😄

**Comment by RobFog** [comment]
Sounds good to me! 👌

**Comment by estella144** [comment]
Yeah... @RobFog and @jas8522 I suppose you are working on that?

**Comment by sjehuda** [comment]
> @Neustradamus What do you need it for?

I assume for advretizing and marketing.

@Neustradamus is a long time well known of free software and communications, namely Jabber/XMPP.

---

As for myself, I'm looking for an SVG icon to be added to [easy-xmpp-invitation](https://github.com/ge0rg/easy-xmpp-invitation).

Is there an SVG logo of Adium, yet?

**Comment by Neustradamus** [comment]
@sjehuda: Thanks for your comment!

I have not found the Adium .SVG logo :/

In more previous links, I have only found:
- https://adium.im/images/logo.png
- https://en.wikipedia.org/wiki/File:Adium.png

It is always needed to create Adium SVG files...
